packages/snowpack/src/commands/build.ts

```diff
@@ -285,16 +285,40 @@ export async function command(commandOptions: CommandOptions) {
     isHmrEnabled: false,
   });
 
-  if (config.buildOptions.minify) {
+  // note: the following logic is a little awkward, but we want to use the same
+  // esbuild service for minification AND/OR source maps
+  if (config.buildOptions.minify || config.buildOptions.sourceMaps) {
```

Comment:
Like the comment here says, the logic here is weird but it’s trying to limit the number of passes we do on the files when it comes to minification and/or source mapping.